stdlib/public/TensorFlow/DataTypes.swift

```diff
@@ -58,17 +55,19 @@ public protocol AccelerableByTensorFlow {
   /// otherwise.  Users should call the _TFGetScalar wrapper function.
   static func _getScalar(_ handle: TensorHandle<Self>) -> Self?
 
-  /// This converts a scalar to a 0d TensorHandle that contains the value.
-  /// Users should call the _TFMakeScalarTensor wrapper function.
-  static func _makeScalarTensor(_ scalar: Self) -> TensorHandle<Self>
-
   /// This indicates that it is safe to hoist the specified computation that
   /// creates a tensor to being a parameter that is passed in from outside of
   /// the tensor program.
   static func _hoistableClosure(_ fn: () -> TensorHandle<Self>)
     -> TensorHandle<Self>
 }
 
+/// A scalar data type compatible with TensorFlow.
+///
+/// Types that conform to `TensorFlowScalar` can be used as the `Scalar`
+/// associated type of `Tensor`.
```

Comment:
Sounds useful for people reading the stdlib implementation. I think it's a bit of an implementation detail that users won't need to see because we never expose the fact that "Scalars" and "Strings" are unified under the hood. I'll add this as a 2-slash comment.

Reply:
Agreed.